docs/components/connectors/use-connectors/inbound.md

```diff
@@ -87,9 +119,65 @@ For example, given that your correlation key is defined with `requestIdValue` pr
+When the process execution arrives at the event gateway, the type of the message determines which path the process will take.
+If each Connector event listened to the message queue using a separate subscription, this might lead to race conditions due to message being received by a different consumer (e.g. `PAYMENT_COMPLETED` event being consumed by the consumer that expects `PAYMENT_CANCELLED`).
+Eventually this might lead to message loss or delayed processing, while also increasing the load on the message broker due to message being returned to the queue.
+
+To avoid this, both events can be assigned to the same subscription by assigning the same **Deduplication ID** to both events. Then, all messages will be consumed by the same subscription, and the Connector runtime will evaluate the **Activation condition** of each event to determine which one should be triggered.
+
+:::note
+When using this pattern, make sure that the **Activation condition** of each event is mutually exclusive, so that only one event is triggered for each message.
+Attempting to trigger multiple events for the same message will result in an error.
+:::
+
+### Automatic deduplication
+
+By default, the Connector runtime will assign the same deduplication ID to Connector events that have equal properties, and different deduplication IDs to events that have different properties.
+The automatic deduplication only takes into account the properties that are related to the business logic of the connector (e.g. **Server URL** or **Authentication properties**).
+It does not take into account the properties that define output mapping (**Result variable**, **Result expression**, **Response expression**), correlation (**Correlation key (process)**, **Correlation key (payload)**, **Activation condition**), or other properties that are handled by the Connector runtime and not by the Connector itself.
+
+This way, two Connectors of the same type that are identical in terms of business logic and are defined in the same business process will be deduplicated automatically.
+
+### Manual deduplication
+
+If the automatic deduplication rules do not work for your use case, you can manually assign a deduplication ID to each Connector event.
+This allows you to group Connectors in a more flexible way, based on your requirements.
+
+If needed, you can have multiple Connectors with the same properties that have different deduplication IDs. This way, you can still have multiple instances of the same Connector listening to the same event source, but each instance will have its own deduplication ID and will be treated as a separate entity by the Connector runtime.
+
+To assign a deduplication ID, you need to do the following:
+
+1. Enable the **Manual mode** checkbox in the **Deduplication** section of the Connector properties.
+2. The **Deduplication ID** field will appear. Fill in the desired deduplication ID.
```

Comment:
It might be obvious, but it would be nice to have an example here

Reply:
I added a screenshot with example input below